pkg/registry/apis/provisioning/repository/github/client.go

```diff
@@ -101,6 +105,19 @@ type RepositoryContent interface {
 	GetSize() int64
 }
 
+type CommitAuthor struct {
+	Name      string
```

Comment:
could we also get username?  eg `ryantxu` vs `Ryan McKinley`

then we can show it in the UI

Reply:
Added: 

![image](https://github.com/user-attachments/assets/03d27390-4c7c-4cc8-b7d5-7e9929143929)